siatest/testnode.go

```diff
@@ -18,6 +18,33 @@ type TestNode struct {
 	primarySeed string
 }
 
+// RestartNode restarts a TestNode
+func (tn *TestNode) RestartNode() error {
+	err := tn.StopNode()
+	if err != nil {
+		return errors.AddContext(err, "Could not stop node")
+	}
+	err = tn.StartNode()
+	if err != nil {
+		return errors.AddContext(err, "Could not start node")
+	}
+	return nil
+}
+
+// StartNode starts a TestNode from an active group
+func (tn *TestNode) StartNode() error {
+	err := tn.LoadNode(tn.params)
```

Comment:
It seems like we only use `LoadNode` here. Why not just move the `LoadNode` code to `StartNode`?

Reply:
I think we also might need a boolean field to check if a node is running or not. Then we can sanity check `StartNode`, `StopNode` and `RestartNode`.